scripts/module.js

```diff
@@ -0,0 +1,287 @@
+  const str = `
+    <section class="popup">
+    <img class="popup-cancel" id="close" src="./assets/IconCancelGray.svg" alt="cancel-icon">
+    <section class="card pop-card">
+        <div class="card-header">
+            <h2>${proj.name}</h2>
+            <ul class="copyright">
+                <li class="m-img">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <img class="m-img" src="${proj.image}" alt="Tonic">
+        <article class="card-container">
+            <p class="content">
+                ${proj.article}
+            </p>
+            <ul class="tags">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">JavaScript</a></li>     
+            </ul>
+            <div class="line">
+                <div></div>
+            </div>
+            <ul class="action">
+                <li>
+                    <a class="action-btn btn" href="${proj.liveLink}">
+                        <p>See live</p>
+                        <img class="action-img" src="./assets/live.svg" alt="live">
+                    </a>
+                </li>
+                <li>
+                    <a class="action-btn btn" href="${proj.sourceLink}">
+                        <p>See source</p>
+                        <img class="action-img" src="./assets/social/githubBlue.svg" alt="source">
+                    </a>
+                </li>
+            </ul>
+        </article>
+    </section>    
+    </section>
+`;
+
+  return str;
+}
+
+const DgeneralArticle = `
+Lorem Ipsum is simply dummy text of the printing and typesetting 
+industry. Lorem Ipsum has been the industry's standard dummy text 
+ever since the 1500s, when an unknown printer took a galley of type 
+and scrambled it 1960s. Lorem Ipsum is simply dummy text of the 
+printing and typesetting industry. 
+<br> <br>
+Lorem Ipsum has been the industry's standard dummy text ever since 
+the 1500s, when an unknown printer took a galley of type and scrambled 
+it 1960s with the releorem Ipsum is simply dummy text of the printing 
+and typesetting industry. Lorem Ipsum han printer took a galley of type 
+and scrambled it 1960s with the releawn printer took a galley of type 
+and scrambled it 1960s.
+`;
+
+const desktopData = [
+  {
+    name: 'Tonic',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    image: './assets/D_cards/D_card1.svg',
+    article: DgeneralArticle,
+    tech: ['html', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '#',
+    sourceLink: 'https://github.com/sagieramos/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    company: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    year: '2015',
+    image: './assets/D_cards/D_card2.svg',
+    article: DgeneralArticle,
+    tech: ['html', 'Ruby on rails', 'CSS', 'JavaScript'],
+    liveLink: '#',
+    sourceLink: 'https://github.com/sagieramos/',
+  },
```

Comment:
- [ ] You did well creating the array of objects of your project info :+1:  but please if you can see have created your different array which makes you repeat yourself here and makes your code plenty and not clean :blush: Why don't you create just one array for all your project details then when you are looping through, you just access the info you what from the project object instead of creating two different arrays while there are many properties of the project object in common?

Reply:
Based on the Figma design of this project (template 1), separate sets of data are intended for the Desktop and Mobile pages.